modules/programs/nushell.nix

```diff
@@ -203,6 +213,26 @@ in {
           cfg.extraLogin
         ];
       })
+
+      (let
+        activateNushellPluginsNuScript = pkgs.writeTextFile {
+          name = "activateNushellPlugins";
+          text = ''
+            #!${pkgs.lib.getExe pkgs.nushell}
```

Comment:
Specifying a shebang is unnecessary if you're passing the program to nushell. Would it be easier to use the form `nu --commands 'plugin add /nix/.../formats; plugin add /nix/.../polars'`?

If not, this can be simplified with

```nix
pkgs.writeText "activateNushellPlugins.nu" ''
  # file contents in here
'';
```

Reply:
Oh, of course.  I had tried using `--commands` instead of generating and passing a script and got a command-not-found error, but your comment prompted me to try again, and it worked, so I must have been doing something else wrong.  I think it looks much tidier now, thanks!